### PR TITLE
bridges | add coyote timer for jumping

### LINKED ISSUE #328: bridges | add coyote time for jumping
## Motivation
Life is harsh, but the game doesn't have to be. It would be nicer if there was a little bit of delay before falling.

## Implementation
Depends on how coyote time is implemented, but it's a common feature and shouldn't take much time.

### PR DESCRIPTION
**Pull request by Kabooni**
## Motivation
closes #328

It would be nice to give the player a small grace period before falling, where jumping is still allowed.

## What was changed/added
Added a timer, which starts when falling and allows jumping once for a duration of `coyote_frames / 60`. I set the frame number to 8 but this can be changed easily in the editor.

## Testing

https://github.com/mango-gremlin/arch-enemies/assets/57258671/691ccdc9-af24-4cfd-83e3-2b8bf552f8c8

### CODE REVIEW

**Review by mango-gremlin — COMMENTED** [review]
works great thanks! I tested it and it worked fine. I didnt test sound though, will do that now, since sound effects were deleted? did you need to commit those files, or was that a mistake?

File: Arch-enemies/bridges/scenes/0_TutorialLevel.tscn [hunk 1]
```diff
@@ -1033,16 +1043,6 @@ scale = Vector2(1, 10)
 position = Vector2(-10, 61)
 scale = Vector2(1, 10)
 
-[node name="goal_menu" parent="Grid" instance=ExtResource("18_65aa1")]
```
[the remark this answers is not in the document]
Comment (Kabooni) [reply Arch-enemies/bridges/scenes/0_TutorialLevel.tscn:1036]: It's not deleted, just re-arranged

File: Arch-enemies/bridges/scenes/0_TutorialLevel.tscn [hunk 2]
```diff
@@ -873,18 +873,28 @@ stream = ExtResource("1_he7ej")
 volume_db = -10.0
 autoplay = true
 
-[node name="Sound Effects" type="AudioStreamPlayer" parent="."]
```
[the remark this answers is not in the document]
Comment (Kabooni) [reply Arch-enemies/bridges/scenes/0_TutorialLevel.tscn:876]: It's not deleted, just re-arranged

File: Arch-enemies/bridges/scenes/0_TutorialLevel.tscn (hunk 1 above)
[the remark this answers is not in the document]
Comment (mango-gremlin) [reply Arch-enemies/bridges/scenes/0_TutorialLevel.tscn:1036]: this probably leads to the following error when trying to finish the tutorial level:
image

File: Arch-enemies/bridges/scenes/0_TutorialLevel.tscn (hunk 2 above)
[the remark this answers is not in the document]
Comment (Kabooni) [reply Arch-enemies/bridges/scenes/0_TutorialLevel.tscn]: I reverted the change, idk when it happened

File: Arch-enemies/bridges/scenes/0_TutorialLevel.tscn (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kabooni) [reply Arch-enemies/bridges/scenes/0_TutorialLevel.tscn]: I reverted the change, I didn't notice it was moved under a node

**Review by marcel-iv — APPROVED** [review]
Tested in all levels, works great. 

Maybe the time is a little bit too forgiving, you can save some animals in each level, which changes balance somewhat. However, thats just a minor thing, and could be changed later on as needed. More importantly, it feels much better than previously.

**Comment by Kabooni** [comment]
After discussions with multiple people, it was decided to reduce the `coyote_frames` number to 6.

**Pull request merged**